includes/wikia/tasks/Tasks/MultiTask.class.php

```diff
@@ -199,19 +259,19 @@ private function runOnWikis($wikiInputRaw, $lang, $cat, $wikiId, $action, $comma
 			$query->AND_( 'city_list.city_lang' )->EQUAL_TO( $lang );
 		}
 
-		if (!empty($wikiId)) {
-			$query->AND_('city_list.city_id')->EQUAL_TO($wikiId);
+		if ( !empty( $wikiId ) ) {
```

Comment:
2. True, but I want to start with delete task. Edit and move can still be executed with no wiki selected. Once we prove this solution works on production, I'll modify the remaining task types.